llvm/test/DebugInfo/CAS/AArch64/debug_line_and_info_dedupe.test

```diff
@@ -5,6 +5,9 @@ RUN: llc -O0 -cas-friendly-debug-info --filetype=obj --cas-backend --cas=%t/cas
 RUN: llc -O0 -cas-friendly-debug-info --filetype=obj --cas-backend --cas=%t/cas --mccas-casid %t/b.ll -o %t/b.id
 RUN: llvm-cas-dump --die-refs --cas=%t/cas --casid-file %t/a.id %t/b.id | FileCheck %s
 
+// LLVM_ENABLE_ONDISK_CAS is not enabled by default on Windows
```

Comment:
The ON_DISK_CAS requirement should probably go into `lit.local.cfg` in the same directory so it doesn't need to be added to all the tests.

This requires should be `ondisk_cas`

Reply:
Done

Reply:
I was actually taking about something like this: https://github.com/swiftlang/llvm-project/blob/next/llvm/test/MC/CAS/lit.local.cfg

This is equivalent to putting `REQURIES: ondisk_cas` on every file in the same directory. I think it will save lots of explicit requirement in the actual test file.